### ISSUE TITLE
Bundling with Webpack fails

### ISSUE THREAD

**Issue by iaxentoi** [dachev/node-cld#67]
We are packaging our project with serverless, which uses Webpack. 
The packaging fails for `cld` with the following error:

`ERROR in ../../cld/index.js 2:13-43
Module not found: Error: Can't resolve './build/Release/cld' in 'node_modules/cld'`

Changing the way it is imported should fix the issue. Is it something that could be implemented?
Thank you!

**Comment by dachev** [comment]
Are you aiming to run this in a browser?

**Comment by iaxentoi** [comment]
No, we are running a set of lambda functions and we package & deploy those using serverless & serverless-serverless-bundle.

**Comment by dachev** [comment]
I'll consider a pull request if you have one but as it stands I don't have time to investigate and fix this.

**Comment by vinaygosain** [comment]
@dachev , is it possible to run this on browser(client side) somehow?

**Comment by dachev** [comment]
@vinaygosain unfortunately this is not possible
